index.html

```diff
@@ -0,0 +1,611 @@
+          <li>
+            <figure class="team-flag">
+              <img src="/img/person-image-placeholder.png" alt="Image of Tom Hasman" title="Tom Hasman">
+              <figcaption>
+                <h2>Tom Hasman</h2>
+                <p>Beta Tester</p>
+              </figcaption>
+            </figure>
+          </li>
+          <li>
+            <figure class="team-flag">
+              <img src="/img/person-image-placeholder.png" alt="Image of Liza Camron" title="Liza Camron">
+              <figcaption>
+                <h2>Liza Camron</h2>
+                <p>Developer</p>
+              </figcaption>
+            </figure>
+          </li>
+        </ul>
+      </section>
+      <section data-section="appbox-featured" id="featured">
+        <h1 class="header-underline">AppBox Featured In ...</h1>
+        <ul class="l-flex-container">
+          <li>
+            <figure>
+              <img src="/img/new-york-times-logo.png" alt="Logo of The New York Times" title="New York Times">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/forbes-logo.png" alt="Logo of Forbes" title="Forbes">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/wall-street-journal-logo.png" alt="Logo of Wall Street Journal" title="Wall Street Journal">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/the-guardian-logo.png" alt="Logo of the Guardian" title="the Guardian">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/verge-logo.png" alt="Logo of The Verge" title="The Verge">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/tech-crunch-logo.png" alt="Logo of TechCrunch" title="TechCrunch">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/lifehacker-logo.png" alt="Logo of lifehacker" title="lifehacker">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/mashable-logo.png" alt="Logo of Mashable" title="Mashable">
+            </figure>
+          </li>
+        </ul>
+      </section>
+      <section data-section="app-support" id="support">
+        <h1>App Support</h1>
+        <div class="l-flex-container-app-support">
+          <ul>
+            <li class="css-flag">
+              <div class="css-flag__image">
+                <span class="icon-location"></span>
+              </div>
+              <div class="css-flag__body">
+                <h2>Address</h2>
+                <p><a href="#">121 King Street, Melbourne<br>Victoria, Australia 3000</a></p>
+              </div>
+            </li>
+            <li class="css-flag">
+              <div class="css-flag__image">
+                <span class="icon-phone"></span>
+              </div>
```

Comment:
In addition to the heretofore unused naming convention, you also are repeating the unnecessary icon markup and classes. Stop it.